pkg/server/bootstrap/identity.go

```diff
@@ -120,7 +120,7 @@ func NewWildcardIdentitiesWrappingRoundTripper(groupExportNames map[string]strin
 		ids.groupResourceIdentities[gr] = ""
 	}
 
-	return injectKcpIdentities(ids), wildcardIdentitiesResolver(ids, groupExportNames, groupResourceExportNames, apiExportIdentityProvider(rootShardKcpClient.Cluster(tenancyv1alpha1.RootCluster), localShardKubeClusterClient))
```

Comment:
I'd prefer that we continue to choose *where* we scope the client - as written, after this change the injection will happen with whatever client is passed in, even though it's only ever valid to inject with the root client. Let's perhaps pass the `*rest.Config` here and create the root client here, as well, to continue to provide that behavior.

Reply:
The client being used here is only scoped to root cluster. This change involves passing a root cluster scoped client instead of an unscoped client to bootstrap. Just wondering, what will be the value of passing config here? If we do so, this would mean that instead of passing clients around to controllers we would be passing configs at places where scoping is possible.